admin/orders.php

```diff
@@ -115,6 +115,14 @@
         $oID = zen_db_prepare_input($_GET['oID']);
         $comments = zen_db_prepare_input($_POST['comments']);
         $status = (int)zen_db_prepare_input($_POST['status']);
+        $order = new order((int)$oID);
+        $emailLanguageQuery = $db->Execute("SELECT directory
+                                        FROM " . TABLE_LANGUAGES . "
+                                        WHERE languages_id = " . (int)$order->info['language_id']);
+        $emailLanguage = $emailLanguageQuery->fields['directory'];
+
+        require(DIR_WS_LANGUAGES . $emailLanguage . '/orders_email.php');
+
```

Comment:
Can I recommend a couple optimizations here?

The query for `$emailLanguageQuery` can be avoided by using something like:
```
$order = new order((intO$oID);
$emailLanguage = $lng->get_language_data_by_id((int)$order->info['language_id']);
zen_load_language_file('orders_email.php', $emailLanguage['directory']);
```

... which depends on also rewriting the `zen_load_language_file()` in `functions_helpers.php` to the following ...
```
/**
 * Lookup specified language file, and load
 *
 * @param string $file filename of language file
 */
function zen_load_language_file($file, $language_directory = '~~') {
    if ($language_directory != '~~') {
      if (file_exists(DIR_WS_LANGUAGES . $language_directory . '/' . $file)) {
        include (DIR_WS_LANGUAGES . $language_directory . '/' . $file);
        return;
      }
    }
    if (file_exists(DIR_WS_LANGUAGES . $_SESSION ['language'] . '/' . $file)) {
        include (DIR_WS_LANGUAGES . $_SESSION ['language'] . '/' . $file);
        return;
    }
    if (file_exists(DIR_WS_LANGUAGES . 'english' . '/' . $file)) {
        include (DIR_WS_LANGUAGES . 'english' . '/' . $file);
    }
}
```